src/main/java/net/sf/jabref/gui/search/SearchResultsDialog.java

```diff
@@ -480,7 +481,7 @@ public Object getColumnValue(BibEntry entry, int column) {
             }
             else {
                 String field = FIELDS[column - PAD];
-                if ("author".equals(field) || "editor".equals(field)) {
+                if (InternalBibtexFields.getFieldExtras(field).contains(BibtexSingleFieldProperties.PERSON_NAMES)) {
```

Comment:
I was considering that, but at the same time there may be quite a few different methods of that type provided that one supports all properties. But I agree that the call is quite long considering that there are long class names etc and that this is probably the most frequent call.